### ISSUE TITLE
Screen size is 1024 x 768 regardless of the viewport setting

### ISSUE THREAD

**Issue by AL-Kateb** [ariya/phantomjs#14231]
Hello, I'm using PhantomJS 2.1.1 on CentOS, I've downloaded the pre-built binary and I'm using it.

The problem is simple, I'm setting `page.viewportSize` to something like 1900 x 600  the screenshot is being rendered correctly 100% and it is indeed 1600 x 600, but when I open a website like http://whatismyscreenresolution.com or http://whatsmyuseragent.com it says my screen resolution is 1024 x 768, I know some might think, if the screenshot is being rendered right what does it matter, but I kind of want those websites to detect the right resolution.

So basically viewportSize is setting the browser's resolution, not the virtual window, is this a bug? Or is there a setting I could use to set the window's resolution.

Thanks

**Comment by Vitallium** [comment]
Yes, you're correct. By default, we simulate typical user screen resolution (https://github.com/Vitallium/qtbase/blob/b5cc0083a5766e773885e8dd624c51a967c17de0/src/plugins/platforms/phantom/phantomintegration.cpp#L94)

PhantomJS is headless browser. It runs usually in environments where you can't determine the screen resolution. So if you can't find the correct resolution - simulate it!
Right now only on Windows PhantomJS reports the correct screen resolution.
